packages/PreviewCard/PreviewCard.jsx

```diff
@@ -31,6 +31,10 @@ const BoxContainer = styled.div`
   display: inline-block;
 `
 
+const HrefTextSmall = styled.span`
+  font-size: 14px;
```

Comment:
Can you use rems instead of pixels for sizes.

Reply:
ok